gradle/multiNodeTesting.gradle

```diff
@@ -1,6 +1,6 @@
 // Actually run the tests by building a cluster of free-running JVMs first.
 task testSingleNode(type: Exec) {
-    dependsOn jar, testJar
+    dependsOn jar, testJar, ":${defaultWebserverModule}:jar"
```

Comment:
It's interesting that for our Gradle `jar` doesn't mean defaultWebserverModule jar.

Reply:
Investigating.

Reply:
testJar is defined as:

```
task testJar(type: Jar, dependsOn: testClasses, group: "build") {
    classifier = 'test'
    from sourceSets.test.output
}
```

Reply:
agree with @Pscheidl that `jar` should already depend on  ":${defaultWebserverModule}:jar" but thats probably not true for all modules

Reply:
That task won't gather any testRuntimeOnly dependencies. testCompile works. I'm not even sure whether this task gets executed.

Reply:
Ok, two observations. This code is a copy&paste of some old stuff usable for old gradles. Obviously, in the resulting classpath, the runtime-only dependencies are not present.

This will fix it:

```
task testJar(type: Jar, dependsOn: testClasses, group: "build") {
    classifier = 'test'
    from sourceSets.test.output + sourceSets.test.runtimeClasspath
}
```

Just add the runtime classpath to the jar and we're done.